ddp-workspace/projects/ddp-testboston/src/app/components/user-registration-prequal/user-registration-prequal.component.scss

```diff
@@ -1,38 +1,39 @@
-.main {
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    min-height: 100%;
-}
-
-.section {
-    width: 100%;
-}
-
-.content {
-    margin: 0 auto;
-    padding: 0 1.5625rem;
-    max-width: 700px;
-}
-
-h1 {
-    font-size: 2rem;
-    line-height: 2.75rem;
-}
-
-p {
-    font-size: 1.375rem;
-    line-height: 2.0625rem;
+.registration-block {
+    margin: 3rem auto 8rem auto;
 }
 
 .invitation-form {
     display: flex;
     flex-direction: column;
 
+    &__title {
+        color: #454545;
+    }
+
```

Comment:
The class name naming is interesting. How did you come up with this? I think it makes a lot of sense.

Reply:
I used BEM methodology and SCSS syntax.